core/src/main/java/com/linecorp/armeria/client/endpoint/StaticEndpointGroup.java

```diff
@@ -56,13 +58,6 @@ public StaticEndpointGroup(Iterable<Endpoint> endpoints) {
 
     @Override
     public String toString() {
-        final StringBuilder buf = new StringBuilder();
-        buf.append("StaticEndpointGroup(");
-        for (Endpoint endpoint : endpoints) {
-            buf.append(endpoint).append(',');
-        }
-        buf.setCharAt(buf.length() - 1, ')');
-
-        return buf.toString();
+        return getClass().getSimpleName() + endpoints;
```

Comment:
Any reason not to use toStringHelper?

Also, since the class is final I guess instead of `getClass()` could be `StaticEndpointGroup.class`

Reply:
> Any reason not to use toStringHelper?

Because it will have an extra pair of curly braces? `StaticEndpointGroup{[...]}` vs `StaticEndpointGroup[...]`